audiomentations/core/composition.py

```diff
@@ -125,20 +143,119 @@ class OneOf(BaseCompose):
     # Augment/transform/perturb the audio data
     augmented_samples = augment(samples=samples, sample_rate=16000)
 
+    # Result: 2 or more transforms will be applied from the list of transforms.
+    ```
+
+    """
+
+    def __init__(
+        self, nbr_transforms: float or tuple, transforms, p: float = 1.0
```

Comment:
I would prefer "num_transforms" over "nbr_transforms", following the naming convention used in other places in audiomentations, like

num_channels
num_frequency_bins
num_semitones
num_places_to_shift

```suggestion
        self, num_transforms: float or tuple, transforms, p: float = 1.0
```

Reply:
And by the way, shouldn't this be `int or tuple` instead of `float or tuple`?

Reply:
Oh yeah, right !